### PR TITLE
PR title modification for a better look

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
@denelon A minor fix. Please approve.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21213)

### CODE REVIEW

File: Tools/YamlCreate.ps1
Comment (Trenly) [inline Tools/YamlCreate.ps1:907]: I would suggest we stay with `version` to match wingetcreate. Commit messages usually are a matter of personal preference anyways, so long as they are understandable and accurate. For most of my commits I like to use `v1.2.3.0`, but there really isn't a difference if it were spelled out
```suggestion
        git commit -m "$CommitType: `$PackageIdentifier` version $PackageVersion"
```

**Comment by denelon** [comment]
@OfficialEsco I'll wait to review/approve/merge until after you validate and approve.

**Review by OfficialEsco — APPROVED** [review]
Yep this did what i actually wanted but i can never find the character when i need it

**Review by denelon — APPROVED** [review]

**Comment by OfficialEsco** [comment]
Well... This is why i had `$CommitType : $PackageIdentifier`
```
PS C:\Users\Esco\GitHub\winget-pkgs> .\Tools\YamlCreate.ps1
ParserError: C:\Users\Esco\GitHub\winget-pkgs\Tools\YamlCreate.ps1:944
Line |
 944 |          git commit -m "$CommitType: `$PackageIdentifier` version $Pac …
     |                         ~~~~~~~~~~~~
     | Variable reference is not valid. ':' was not followed by a valid variable name character. Consider
     | using ${} to delimit the name.
```

**Comment by OfficialEsco** [comment]
Fixed with `git commit -m "$CommitType: `$PackageIdentifier` version $PackageVersion"`
but then this happened #21220
Not sure if this is on my end
image

**Comment by vedantmgoyal2009** [comment]
I am sorry, it should be: ```git commit -m "$CommitType`: ``$PackageIdentifier`` version $PackageVersion```
This one is tried and tested, so I am very much sure, it will work without throwing any error. Here, it writes

---------------------------------------------
image
---------------------------------------------
**... what should be entered here...**

---------------------------------------------
image
---------------------------------------------

**Pull request merged**
